src/test/java/com/google/api/codegen/transformer/go/GoModelTypeNameConverterTest.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.go;
+
+import com.google.api.codegen.util.TypeName;
+import com.google.common.truth.Truth;
+
+import org.junit.Test;
+
+public class GoModelTypeNameConverterTest {
+
+  private static final GoModelTypeNameConverter converter = new GoModelTypeNameConverter();
+
+  @Test
+  public void testGetTypeName() {
+    // Not in curated proto. Don't guess anything.
+    Truth.assertThat(
+            converter.getTypeName("github.com/someone/repo", "foo.bar", "Baz", false).getNickname())
```

Comment:
to make the test more readable (w.r.t. the booleans), could you make two "helper" variables, isPointerTrue and isPointerFalse, and pass them to getTypeName instead of true/false?